src/CppUTest/SimpleString.cpp

```diff
@@ -490,6 +490,21 @@ SimpleString HexStringFrom(const void* value)
     return StringFromFormat("%lx", convertPointerToLongValue(value));
 }
 
+SimpleString StringFrom(const unsigned char* value, size_t size)
+{
+    SimpleString str = StringFromFormat("Len = %lu | HexContents =", size);
+    size_t displayedSize = ((size > 128) ? 128 : size);
+    for (size_t i = 0; i < displayedSize; i++)
+    {
+        str += StringFromFormat(" %02X", value[i]);
+    }
+    if( size > displayedSize )
+    {
+        str += " ...";
+    }
+    return str;
+}
+
```

Comment:
Yeah, thinking about it, I guess the printing of the length of the buffer maybe shouldn't be in the StringFrom?